src/pages/LandingPage.js

```diff
@@ -70,6 +70,10 @@ const LandingPage = pure(() => {
         ]),
         makeCard('library-showcase', 'View Examples', 'Browse our gallery of showcase Workspaces to see how science gets done on Terra.'),
         makeCard('library-datasets', 'Browse Data', 'Access data from a rich ecosystem of Terra-connected data portals.')
+      ]),
+      (isTerra() || isFirecloud()) && div({ style: { width: 700, marginTop: '4rem' } }, [
```

Comment:
nit: Minor divergence from the mock - should the text be just above the footer?

Reply:
Good point. I think we should move ahead with this for now, since pushing it to the bottom would require a bit more restructuring of the page, and we've got several changes in flight that might conflict. We can follow up with tweaks afterward.